File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0)
            return false;
        else{
        int num = x; 
        long sum = 0, n;
        while(x != 0){
            n = x % 10;
            sum = 10 * sum + n;
            x = x / 10;
        }
        if(sum == num){
            return true;
        }
        else{
            return false;
        }
    }
    }
};

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x<0:
            return False
        else:
            num = x
            sum = 0
            while x!=0:
                dgt = x%10
                sum = 10*sum + dgt
                x = x//10
        
        if sum==num:
            return True
        else:
            return False